lib/irc.js

```diff
@@ -654,6 +655,9 @@ Client.prototype.connect = function(retryCount, callback) { // {{{
                     self.conn.authorizationError === 'CERT_HAS_EXPIRED') {
                     util.log('Connecting to server with expired certificate');
                 }
+                if (typeof self.opt.webirc.ip !== 'undefined') {
```

Comment:
this still has the issue of the case where `self.opt.webirc.ip === ''`. It is probably better to check the value instead of trusting the type. Using `if (self.opt.webirc.ip) {` looks for a falsy value, and the empty string(`''`) will evaluate to `false`.